Jenkinsfile

```diff
@@ -31,6 +60,14 @@ def pytorch_unit_test(dev) {
   }
 }
 
+def pytorch_unit_test_win64(dev) {
+  withEnv(["DGL_LIBRARY_PATH=${env.WORKSPACE}\\build", "PYTHONPATH=${env.WORKSPACE}\\python"]) {
+    bat "python -m nose -v --with-xunit tests"
+    bat "python -m nose -v --with-xunit tests\\pytorch"
+    bat "python -m nose -v --with-xunit tests\\graph_index"
+  }
+}
```

Comment:
Similar to above. How about having `task_unit_tests.sh` and `task_unit_test.bat`? They can accept device and backend as arguments.